packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -49,21 +48,28 @@ class SliderTheme extends InheritedWidget {
   /// Defaults to the ambient [ThemeData.sliderTheme] if there is no
   /// [SliderTheme] in the given build context.
   ///
-  /// Typical usage is as follows:
+  /// ## Sample code
   ///
   /// ```dart
-  /// double _rocketThrust;
-  ///
-  /// @override
-  /// Widget build(BuildContext context) {
-  ///   return new SliderTheme(
-  ///     data: SliderTheme.of(context).copyWith(activeRail: Colors.orange),
-  ///     child: new Slider(
-  ///       onChanged: (double value) => setState(() => _rocketThrust = value),
-  ///       value: _rocketThrust;
-  ///     ),
-  ///   );
+  /// class Catatonic extends StatefulWidget {
+  ///   @override
+  ///   State createState() => new CatatonicState();
   /// }
+  /// 
+  /// class CatatonicState extends State<Catatonic> {
```

Comment:
CatatonicState can be private.

In an abundance of caution, I would recommend using the rocket thrust example rather than one relating to cataonia.

Reply:
OK.  I was just trying to think of "state" puns.